pandas/io/formats/format.py

```diff
@@ -1956,18 +1956,22 @@ def __call__(self, num: int | float) -> str:
         """
         Formats a number in engineering notation, appending a letter
         representing the power of 1000 of the original number. Some examples:
-
-        >>> format_eng(0)       # for self.accuracy = 0
+        >>> format_eng = EngFormatter(accuracy=0, use_eng_prefix=True)
+        >>> format_eng(0)
         ' 0'
-
-        >>> format_eng(1000000) # for self.accuracy = 1,
-                                #     self.use_eng_prefix = True
+        >>> format_eng = EngFormatter(accuracy=1, use_eng_prefix=True )
```

Comment:
Can you remove the space after True, that's also against Python styling.

Reply:
I wonder why pep8speaks didn't flag this.